pkg/kubelet/kubelet.go

```diff
@@ -543,7 +546,9 @@ func (kl *Kubelet) GetNode() (*api.Node, error) {
 		return nil, errors.New("cannot list nodes")
 	}
 	host := kl.GetHostname()
+	glog.V(2).Infof("hostname: %q", host)
```

Comment:
Remove these two debug lines? If you want to keep the hostname one can we make it V(4)?